test/webaudio/audioworklet.c

```diff
@@ -110,6 +111,7 @@ void AudioWorkletProcessorCreated(EMSCRIPTEN_WEBAUDIO_T audioContext, bool succe
 // Processors.
 void WebAudioWorkletThreadInitialized(EMSCRIPTEN_WEBAUDIO_T audioContext, bool success, void *userData) {
   assert(success && "Stopped in WebAudioWorkletThreadInitialized");
+  printf("audio worklet initialzied\n");
```

Comment:
Spelling of 'initialzied' (and you've no idea the finger pain I went through to write that with a zed...).